frontend/src/Console.jsx

```diff
@@ -36,26 +47,70 @@ function Console(props) {
   }
 
   function toggleModal() {
-    
     setModal({ isOpen: !modalState.isOpen });
   }
 
+  function sendForm(formDetails) {
+    if (formInfo.postOrPut === "post") {
+      apiUtil[formInfo.postOrPut](
+        `/${formInfo.entityName}`,
+        formDetails,
+        (data) => {
+          setDishes([data]);
+          console.log(data);
+        },
+        (err) => {
+          console.log(err);
+        }
+      );
+    } else {
+      apiUtil[formInfo.postOrPut](
+        `/${formInfo.entityName}/${formInfo.entity._id}`,
+        formDetails,
+        (data) => {
+          setDishes([data]);
+          console.log(data);
+        },
+        (err) => {
+          console.log(err);
+        }
+      );
+    }
+  }
+
+  function toggleLanguage() {
+    if (language === "hebrew") {
```

Comment:
Same comment as before - always try not to have specific conditions.
Have a validation on the language and then set it in a generic way.
Also - toggle is not smart here because then when we'll add more languages this logic won't work.
Do it generically, send a parameter that indicates the selected language

Reply:
I changed it to be change language with validation that the language is supported